presto-main/src/test/java/com/facebook/presto/operator/aggregation/AbstractTestApproximateCountDistinct.java

```diff
@@ -142,7 +142,7 @@ private long estimateGroupByCount(List<Object> values, double maxStandardError)
         return (long) result;
     }
 
-    private long estimateCount(List<Object> values, double maxStandardError)
+    protected long estimateCount(List<Object> values, double maxStandardError)
```

Comment:
Instead of making this protected, i would make `com.facebook.presto.operator.aggregation.AbstractTestApproximateCountDistinct#assertCount` protected.